src/Image/index.tsx

```diff
@@ -1,6 +1,17 @@
+import { Image as ImageNative } from 'react-native';
 import { withTheme } from '../config';
 import { Image, ImageProps } from './Image';
 
 export { Image };
 export type { ImageProps };
-export default withTheme(Image, 'Image');
+
+const ThemedImage = Object.assign(withTheme(Image, 'Image'), {
+  getSize: ImageNative.getSize,
+  getSizeWithHeaders: ImageNative.getSizeWithHeaders,
+  prefetch: ImageNative.prefetch,
+  abortPrefetch: ImageNative.abortPrefetch,
+  queryCache: ImageNative.queryCache,
+  resolveAssetSource: ImageNative.resolveAssetSource,
+});
+
```

Comment:
Can we do something like this 🤔 

```js
{
...withTheme(Image, 'Image'),
  getSize: ImageNative.getSize,
  getSizeWithHeaders: ImageNative.getSizeWithHeaders,
  prefetch: ImageNative.prefetch,
  abortPrefetch: ImageNative.abortPrefetch,
  queryCache: ImageNative.queryCache,
  resolveAssetSource: ImageNative.resolveAssetSource,
}
```

Reply:
`JSX element type 'Image' does not have any construct or call signatures.`